src/pages/lesson-plans/lesson-06-notes.md

```diff
@@ -3,3 +3,19 @@
 - fix `<ol>` stuff in `8. Closure`
 - remove `<step>` before `Students may try to get up`
 - ratings placeholders
```

Comment:
this is where I took notes on the things I had to tweak manually after the main transformation